desc/profiles.py

```diff
@@ -42,14 +44,26 @@ def name(self, new):
         self._name = new
 
     @property
-    @abstractmethod
     def grid(self):
         """Default grid for computation."""
+        return self._grid
 
     @grid.setter
-    @abstractmethod
-    def grid(self, new):
-        """Set default grid for computation."""
+    def grid(self, grid):
+        if isinstance(grid, Grid):
+            self._grid = grid
+            return
+        if np.isscalar(grid):
+            grid = np.linspace(0, 1, grid)
+            grid = np.atleast_1d(grid)
```

Comment:
Why the call to `np.atleast_1d`? think `linspace` returns minimum dimension 1

Reply:
yeah good point